cpu/samd5x/include/candev_samd5x.h

```diff
@@ -169,11 +169,9 @@ typedef struct {
     /** CAN device configuration descriptor */
     const can_conf_t *conf;
     /** CAN message RAM accessible to the CAN controller */
-    msg_ram_conf_t msg_ram_conf;
+    can_msg_ram_t msg_ram;
     /** Enable/Disable Transceiver Delay Compensation */
     bool tdc_ctrl;
-    /** False to use Tx FIFO operation, True to use Tx Queue operation */
-    bool tx_fifo_queue_ctrl;
```

Comment:
this is unused (unimplemented), non common ( not found in othe candrivers -> not a compatibility thing) 

remove it so no one expects this to work in one way or another.